sdk/storage/storage-blob/samples/javascript/iterators-without-await.js

```diff
@@ -0,0 +1,61 @@
+/*
+ Setup: Enter your storage account name and shared key in main()
+*/
+
+const { BlobServiceClient, StorageSharedKeyCredential } = require("../../"); // Change to "@azure/storage-blob" in your package
+
+async function main() {
+  // Enter your storage account name and shared key
+  const account = process.env.ACCOUNT_NAME || "";
+  const accountKey = process.env.ACCOUNT_KEY || "";
+
+  // Use StorageSharedKeyCredential with storage account and account key
+  // StorageSharedKeyCredential is only avaiable in Node.js runtime, not in browsers
+  const sharedKeyCredential = new StorageSharedKeyCredential(account, accountKey);
+
+  const blobServiceClient = new BlobServiceClient(
+    `https://${account}.blob.core.windows.net`,
+    sharedKeyCredential
+  );
+
+  // Create a container
+  const containerName = `newcontainer${new Date().getTime()}`;
+  const containerClient = blobServiceClient.getContainerClient(containerName);
+
+  const createContainerResponse = await containerClient.create();
+  console.log(`Created container ${containerName} successfully`, createContainerResponse.requestId);
+
+  for (let index = 0; index < 7; index++) {
```

Comment:
feels kinda weird that we use `index` up here, where the loop variable isn't important, but we use `i` down below where it is. Maybe flip these?

Reply:
flipped